From b46f730c87487b5b89191860f43e7faa3b33e3b1 Mon Sep 17 00:00:00 2001
From: Jean Christophe Beyler <jean.christophe.beyler@intel.com>
Date: Tue, 21 May 2013 09:49:15 -0700
Subject: Dalvik: Removing Compiler Warnings Part 2

BZ: 109746

Removing a compiler warning about an initialization field.

Category: device-enablement
Domain: AOSP-Dalvik-Compiler-CG
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: Ic5d1bce3daeafde334a11904db1e7b49cd4b4cf1
Orig-MCG-Change-Id: I5418edbb1236f45a4a65b8394fa6aee2a1bb778a
Signed-off-by: Jean Christophe Beyler <jean.christophe.beyler@intel.com>
Signed-off-by: Qiming Shi <qiming.shi@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/compiler/codegen/x86/CodegenInterface.cpp |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/vm/compiler/codegen/x86/CodegenInterface.cpp b/vm/compiler/codegen/x86/CodegenInterface.cpp
index 828118a..e23e042 100644
--- a/vm/compiler/codegen/x86/CodegenInterface.cpp
+++ b/vm/compiler/codegen/x86/CodegenInterface.cpp
@@ -64,7 +64,9 @@ static void SendUpdateToVTune(void * address, unsigned size, unsigned method_id
         return;
     }
 
-    iJIT_Method_Load jitMethod = {0};
+    iJIT_Method_Load jitMethod;
+    memset (&jitMethod, 0, sizeof (jitMethod));
+
     jitMethod.method_id = method_id;
     jitMethod.method_load_address = address;
     jitMethod.method_size = size;
-- 
1.7.4.1

